core/dynamic-tags/manager.php

```diff
@@ -278,19 +275,18 @@ public function get_tags() {
 	}
 
 	/**
-	 * @since 2.0.0
-	 * @access public
+	 * @param string $class
 	 * @deprecated 3.5.0 Use `$this->register()` instead.
 	 *
-	 * @param string $class
+	 * @since 2.0.0
+	 * @access public
```

Comment:
@davseve why replacing the order?

Reply:
This made automatically, 
I thinks that my PHPCS is not defined as it should be.
working on it

Reply:
fixed